site/docs/contribute/environment.md

```diff
@@ -46,3 +46,36 @@ a docker registry as a proxy for all public images. This registry runs in a dock
 export DOCKERHUB_USERNAME='your docker hub username'
 export DOCKERHUB_PASSWORD='your password'
 ```
+
+## Optional Code Improvement Tools 
```

Comment:
Can you change it to `Optional Code Improvement and Verification Tools`

After this high line, please add some explanation, that the Fybrik repositories use different linters to validate and improve code. That `golangci-lint` works as a lints umbrella, and it runs tens different linters, for example ....
That you can see  `golangci-lint` configuration in  `.golangci.yml`